src/components/EditImageModal/EditImageModal.test.jsx

```diff
@@ -576,7 +623,7 @@ describe('EditImageModal', () => {
       it('width responds to height change', () => {
         imageDimensionsHeightInput.find('input').simulate('change', { target: { value: '400' } });
 
-        updateConsants();
+        updateConstants();
 
         // issue with props in asInput expecting a string even if type is number
```

Comment:
I just noticed this comment I think I left. I think this isn't an issue anymore, right? Should we remove this comment?

Reply:
I changed the values in the tests with this comment to numbers and the tests pass with no errors/warnings, so I think it is resolved. I deleted these comments.